sympy/stats/error_prop.py

```diff
@@ -0,0 +1,62 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
+
+
+def variance_prop(expr, consts=()):
+    """Symbolically propagates variance (sigma^2) for expressions.
```

Comment:
Can you give a mathematical definition of variance propagation? Also list any statistical assumptions that are made. A reference to Wikipedia and/or mathworld or any other good reference would also be nice (you can search for examples of how to format references from other docstrings).